Client/Utils/HistoryDeletionUtility.swift

```diff
@@ -102,36 +99,13 @@ class HistoryDeletionUtility: HistoryDeletionProtocol {
 
         switch dateOption {
         case .allTime:
-            switch profile.historyApiConfiguration {
-            case .old:
-                profile.history
-                    .clearHistory()
-                    .uponQueue(.global(qos: .userInteractive)) { result in
-                        completion(result.isSuccess)
-                    }
-            case .new:
-                _ = profile.history
-                    .clearHistory()
-                profile.places.deleteEverythingHistory().uponQueue(.global(qos: .userInteractive)) { result in
-                    completion(result.isSuccess)
-                }
+            profile.places.deleteEverythingHistory().uponQueue(.global(qos: .userInteractive)) { result in
+                completion(result.isSuccess)
             }
         default:
             guard let date = dateFor(dateOption) else { return }
-            switch profile.historyApiConfiguration {
-            case .old:
-                profile.history
-                    .removeHistoryFromDate(date)
-                    .uponQueue(.global(qos: .userInteractive)) { result in
-                        completion(result.isSuccess)
-                    }
-            case .new:
-                // for now we also delete the old history to keep the browser.db up to date
-                _ = profile.history
-                    .removeHistoryFromDate(date)
-                profile.places.deleteVisitsBetween(date).uponQueue(.global(qos: .userInteractive)) { result in
-                    completion(result.isSuccess)
-                }
+            profile.places.deleteVisitsBetween(date).uponQueue(.global(qos: .userInteractive)) { result in
```

Comment:
@OrlaM's question about queue priority applies here too I think. Why .userInteractive and not default?

Reply:
ditto on me not changing this, but maybe I should?